tests/integration-tests/tests/schedulers/test_slurm/test_scontrol_reboot/pcluster.config.yaml

```diff
@@ -0,0 +1,24 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+Scheduling:
+  Scheduler: {{ scheduler }}
+  {{ scheduler_prefix }}Queues:
+    - Name: queue1
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+      ComputeResources:
+        - Name: t2medium
+          InstanceType: t2.medium
```

Comment:
Why not using a t2.micro (4x cheaper) or a t2.small (2x cheaper) for this type of test?